--- a/Setup
+++ b/Setup
@@ -51,7 +51,7 @@
 mouse src_c/mouse.c $(SDL) $(DEBUG)
 rect src_c/rect.c $(SDL) $(DEBUG)
 rwobject src_c/rwobject.c $(SDL) $(DEBUG)
-surface src_c/surface.c src_c/alphablit.c src_c/surface_fill.c $(SDL) $(DEBUG)
+surface src_c/simd_blitters_sse2.c src_c/simd_blitters_avx2.c src_c/surface.c src_c/alphablit.c src_c/surface_fill.c $(SDL) $(DEBUG) -DPG_ENABLE_ARM_NEON
 surflock src_c/surflock.c $(SDL) $(DEBUG)
 time src_c/time.c $(SDL) $(DEBUG)
 joystick src_c/joystick.c $(SDL) $(DEBUG)
